### DISCUSSION TITLE
How to transfer CHIA mainnet DB(now up to 33GB) to another disk?

### DISCUSSION THREAD

**Discussion by wc357** [Chia-Network/chia-blockchain#7139] [Support]
The default installation file of CHIA mainnet DB is getting bigger, sooner or later will run out of C disk space, how to transfer DB to another disk? Or don't install CHIA mainnet DB on C disk at all?

**Comment by woodycal** [comment]
If your windows go to c drive users then your user. look for .chia then mainnet db and copy blockchain_v1_mainnet.sqlite to usb whatever. You can also copy peer_table_node.sqlite this will help with finding peers but can be buggy. When you transfer to another desktop make sure you delete the above files before you paste.

Also like to mention you will need to also copy wallet db to use wallet otherwise that could also take a while to sync. same as above but go wallet then db and you should see two files again a main db one and a peer one. copy paste and make sure you delete any others in new desktop.

Also make sure all chia clients are closed not running in background.

**Reply by wc357** [reply]
THANKS.I have tried to copy the mainnet DB to "D: \\" and log in again to find the maintDB on disk "D: \\" in chia, but chia still creates a new file in C: \Users\Administrator and starts a new sync.It looks like the mainnetDB is bound to the C:\ Users\Administrator.

**Reply by woodycal** [reply]
If your trying to move to another drive on same pc im not sure how you do this at least with windows. I haven't seen an option for this in chia windows where you can change the directory for db files. I assume by how little your drive is your running an os from a usb stick or something?.

**Reply by wc357** [reply]
After rebooting, I reconfirmed the size of mainnetDB space again and it reduced to 6G size,it's not so urgent for now.My OS is installed on a 128GB SSD.Thanks

**Comment by max918** [comment]
why your mainnet DB file have 33G ?
in my full node server it just 6.4G

**Reply by wc357** [reply]
Could it be because I've been running 7/24 for weeks? After rebooting, I reconfirmed the size of mainnetDB space again and it reduced to 6G size

**Comment by loppefaaret** [comment]
the env var `CHIA_ROOT` can be defined, and it will control the location ofthe working dir used for Chia

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
